### PR TITLE
feat(cmd/network/show): Add cmd for fetching compute committees

### LINKED ISSUE #104: Add command for fetching compute committees
Currently, you can do `oasis net show nodes`, but you cannot determine which of those are actually elected in the (compute) committee for each ParaTime. Add something like `oasis net show committee <ParaTimeID>` which dumps the list of nodes in the given compute committee (and optionally for the specific block height).

### PR DESCRIPTION
**Pull request by amela**
Fixes #104

### CODE REVIEW

File: cmd/network/show.go [hunk 1]
```diff
@@ -56,6 +58,11 @@ var showCmd = &cobra.Command{
 		consensusConn := conn.Consensus()
 		registryConn := consensusConn.Registry()
 
+		ctrlConn := conn.Control()
+		nodeStatus, err := ctrlConn.GetStatus(ctx)
```
Comment (kostko) [inline cmd/network/show.go:62]: Why are you getting committees from the node status? Then you are limited only to the runtimes that a given node has configured while other runtimes may exist. In order to support all runtimes, you could get a list of all runtimes from the registry via `registryConn.GetRuntimes(...)`. And you could skip any suspended runtimes that don't have committees.

File: cmd/network/show.go [hunk 2]
```diff
@@ -195,6 +202,39 @@ var showCmd = &cobra.Command{
...
+						for i, member := range state.Committee.Members {
+							nodeQuery := &registry.IDQuery{
+								Height: height,
+								ID:     member.PublicKey,
+							}
+
+							node, err := consensusConn.Registry().GetNode(ctx, nodeQuery)
+							cobra.CheckErr(err)
+
+							fmt.Printf("%2d, entityID: %s, nodeID: %s, role: %s\n", i, node.EntityID, member.PublicKey.String(), member.Role.String())
```
Comment (kostko) [inline cmd/network/show.go:231]: Could you output this in a table form? We already have support for outputting tables.

File: cmd/network/show.go [hunk 3]
```diff
@@ -33,10 +34,11 @@ const (
 	selValidators
 	selNativeToken
 	selGasCosts
+	selCommittee
 )
 
 var showCmd = &cobra.Command{
-	Use:     "show { <id> | entities | nodes | paratimes | validators | native-token | gas-costs }",
+	Use:     "show { <id> | entities | nodes | paratimes | validators | native-token | gas-costs | committee }",
```
Comment (kostko) [inline cmd/network/show.go:41]:
```suggestion
	Use:     "show { <id> | entities | nodes | paratimes | validators | native-token | gas-costs | committees }",
```

File: cmd/network/show.go [hunk 4]
```diff
@@ -33,10 +34,11 @@ const (
 	selValidators
 	selNativeToken
 	selGasCosts
+	selCommittee
```
Comment (kostko) [inline cmd/network/show.go:37]:
```suggestion
	selCommittees
```

File: cmd/network/show.go [hunk 5]
```diff
@@ -242,6 +282,8 @@ func selectorFromString(s string) propertySelector {
 		return selNativeToken
 	case "gas-costs":
 		return selGasCosts
+	case "committee":
```
Comment (kostko) [inline cmd/network/show.go:285]:
```suggestion
	case "committees":
```

File: cmd/network/show.go (hunk 1 above)
Comment (amela) [reply cmd/network/show.go]: Ok, now I am getting a list of all runtimes from the registry, and runtime state from `roothashConn`. Hope it is correct.

File: cmd/network/show.go [hunk 6]
```diff
@@ -195,6 +199,58 @@ var showCmd = &cobra.Command{
...
+				})
+				cobra.CheckErr(err)
+
+				for _, runtime := range runtimes {
+					// generic compute runtime is 1
+					if runtime.Kind != 1 {
+						continue
+					}
+					table := table.New()
+					table.SetHeader([]string{"EntityID", "NodeID", "Role"})
```
Comment (kostko) [inline cmd/network/show.go:215]:
```suggestion
					table.SetHeader([]string{"Entity ID", "Node ID", "Role"})
```

File: examples/network-show/committee.in.static [hunk 7]
```diff
@@ -0,0 +1 @@
+oasis network show committee
```
Comment (kostko) [inline examples/network-show/committee.in.static:1]:
```suggestion
oasis network show committees
```

File: docs/network.md [hunk 8]
```diff
@@ -324,6 +324,14 @@ Above, we can see that the [maximum amount of gas](./account.md#gas-limit) our
 transaction can spend must be set to at least 1000 **gas units**, otherwise it
 will be rejected by the network.
 
+#### `committee` {#committee}
```
Comment (kostko) [inline docs/network.md:327]:
```suggestion
#### `committees` {#committees}
```

File: cmd/network/show.go [hunk 9]
```diff
@@ -195,6 +199,58 @@ var showCmd = &cobra.Command{
...
+			case selCommittees:
+				runtimes, err := registryConn.GetRuntimes(ctx, &registry.GetRuntimesQuery{
+					Height:           height,
+					IncludeSuspended: false,
+				})
+				cobra.CheckErr(err)
+
+				for _, runtime := range runtimes {
+					// generic compute runtime is 1
+					if runtime.Kind != 1 {
```
Comment (kostko) [inline cmd/network/show.go:211]: Use a constant here, e.g. `registry.KindCompute`.

**Review by kostko — APPROVED** [review]

**Pull request merged**